machine_data_model/nodes/connectors/opcua_connector.py

```
@@ -0,0 +1,440 @@
+                    certificate=self.certificate_file_path,
+                    private_key=self.private_key_file_path,
+                    server_certificate=None,  # "certificate-example.der",
+                )
+            except (ConnectionRefusedError, TimeoutError) as e:
+                _logger.error(e)
+                return False
+
+        # TODO: handle trust store
+        if USE_TRUST_STORE:
+            trust_store = TrustStore(
+                [Path("examples") / "certificates" / "trusted" / "certs"], []
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)